readmes/README.Linux.md

```diff
@@ -66,12 +76,32 @@
   sudo ./README.Ubuntu.bash
   ```
 
+- If the script download fails, you can try changing the software download source for Ubuntu
+
+    ```bash
+    cp /etc/apt/sources.list /etc/apt/sources.list.bak_yyyymmdd
+    vim /etc/apt/sources.list #Open the sources.list file
+    
+    #Add the following content to the end of the file
+    deb http://mirrors.aliyun.com/ubuntu/ bionic main restricted universe multiverse
+    deb http://mirrors.aliyun.com/ubuntu/ bionic-security main restricted universe multiverse
+    deb http://mirrors.aliyun.com/ubuntu/ bionic-updates main restricted universe multiverse
+    deb http://mirrors.aliyun.com/ubuntu/ bionic-proposed main restricted universe multiverse
+    deb http://mirrors.aliyun.com/ubuntu/ bionic-backports main restricted universe multiverse
+    deb-src http://mirrors.aliyun.com/ubuntu/ bionic main restricted universe multiverse
+    deb-src http://mirrors.aliyun.com/ubuntu/ bionic-security main restricted universe multiverse
+    deb-src http://mirrors.aliyun.com/ubuntu/ bionic-updates main restricted universe multiverse
+    deb-src http://mirrors.aliyun.com/ubuntu/ bionic-proposed main restricted universe multiverse
+    deb-src http://mirrors.aliyun.com/ubuntu/ bionic-backports main restricted universe multiverse
+    ```
 - Ubuntu 18.04 and newer should have use gcc 7 or newer, but you can also enable gcc-7 on older versions of Ubuntu:
 
   ```bash
-  sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
-  sudo apt-get update
-  sudo apt-get install -y gcc-7 g++-7
+  ## Installing gcc-9
+  sudo apt install software-properties-common
+  sudo add-apt-repository ppa:ubuntu-toolchain-r/test
+  sudo apt install gcc-9 g++-9
+  sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-9 100
   ```
```

Comment:
What GCC version should we use?  GCC 7 or GCC 9? The description has conflicts.

Reply:
After verification, in order to maintain the consistency of the gcc version, the platform uniformly uses gcc-10, and the relevant documents will be changed